db/migrate/20150611182328_add_songs_count_to_artists.rb

```diff
@@ -0,0 +1,7 @@
+class AddSongsCountToArtists < ActiveRecord::Migration
+
+  def change
+    add_column :artists, :songs_count, :integer, :null => false, :default => 0
```

Comment:
Unless necessary (which happens sometimes when dealing this hashes of string keys), we avoid the `=>` and just do:

`null: false, default: 0`

Often when we use gems, they don't have the same guidelines as we do and have to clean up after them when they auto generate migrations like perhaps counter culture did here.